deploy/kubernetes/auditbeat/auditbeat-daemonset.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       - name: auditbeat
         image: docker.elastic.co/beats/auditbeat:%VERSION%
         args: [
-          "-c", "/etc/auditbeat.yml"
+          "-c", "/etc/auditbeat.yml",
```

Comment:
Can you also update

https://github.com/elastic/beats/blob/bc044bd40731f108694595627f420c2a81dd6fbb/deploy/kubernetes/auditbeat-kubernetes.yaml#L94